docs/Makefile

```diff
@@ -1,25 +1,20 @@
 HUGO?=hugo
-# the officially recommended unofficial docker image
 HUGO_IMG?=hugomods/hugo:0.115.3
 
 THEME_MODULE = github.com/nginxinc/nginx-hugo-theme
-## Pulls the current theme version from the Netlify settings
 THEME_VERSION = 0.41.14
-NETLIFY_DEPLOY_URL = ${DEPLOY_PRIME_URL}
 
-# if there's no local hugo, fallback to docker
 ifeq (, $(shell ${HUGO} version 2> /dev/null))
 ifeq (, $(shell docker version 2> /dev/null))
     $(error Docker and Hugo are not installed. Hugo (<0.91) or Docker are required to build the local preview.)
 else
-    HUGO=docker run --rm -it -v ${CURDIR}:/src -p 1313:1313 ${HUGO_IMG} hugo
+    HUGO=docker run --rm -it -v ${CURDIR}:/src -p 1313:1313 ${HUGO_IMG} hugo --bind 0.0.0.0 -p 1313
 endif
 endif
 
 MARKDOWNLINT?=markdownlint
 MARKDOWNLINT_IMG?=ghcr.io/igorshubovych/markdownlint-cli:latest
 
-# if there's no local markdownlint, fallback to docker
 ifeq (, $(shell ${MARKDOWNLINT} version 2> /dev/null))
 ifeq (, $(shell docker version 2> /dev/null))
 ifneq (, $(shell $(NETLIFY) "true"))
```

Comment:
is this if block still required without netlify?

Reply:
Yes, because the Docker fallback has nothing to do with Netlify and is a convenience for a user to not have to install anything on their machines.

Reply:
Sorry, I'm confused.  
```
ifneq (, $(shell $(NETLIFY) "true"))
```
has nothing to do with netlify?

Reply:
Oh, I didn't notice you were referring to a specific line: I thought you were asking about the block as a whole.

Removing those.